source/beamline/AMPVControl.cpp

```diff
@@ -249,6 +250,13 @@ AMControl::FailureExplanation AMPVControl::move(double setpoint) {
 			return NotConnectedFailure;
 		}
 		setpoint_ = setpoint;
+
+		// Update the move progress values.
+
+		updateMoveStart();
```

Comment:
That's a good idea. I was wondering, though, does it make sense for the three values (start, value, end) to be implemented in AMControl? Or do they more logically belong to a subclass?